spec/bundler/definition_spec.rb

```diff
@@ -6,7 +6,7 @@
   before do
     allow(Bundler).to receive(:settings) { Bundler::Settings.new(".") }
     allow(Bundler).to receive(:default_gemfile) { Pathname.new("Gemfile") }
-    allow(Bundler).to receive(:ui) { double("UI", :info => "") }
+    allow(Bundler).to receive(:ui) { double("UI", :info => "", :debug => "") }
```

Comment:
We're adding debug messages so we need to update the double for that expectation.